cmd/ipfs-cluster-service/main.go

```diff
@@ -358,88 +361,103 @@ prints the state to stdout.
 						}
 						defer w.Close()
 
-						err = export(ctx, w)
-						checkErr("exporting state", err)
+						cfgMgr, cfgs := makeAndLoadConfigs()
+						defer cfgMgr.Shutdown()
+						mgr := newStateManager(c.String("consensus"), cfgs)
+						checkErr("exporting state", mgr.ExportState(w))
+						logger.Info("state successfully exported")
```

Comment:
We need to close the state when done with all these operations (badger specially).